@baldersheim Note that some config-model unit test must be updated:
`java.lang.AssertionError: expected:<MMAP> but was:<DIRECTIO>
	at com.yahoo.vespa.model.search.NodeFlavorTuningTest.assertSummaryReadIo(NodeFlavorTuningTest.java:115)
	at com.yahoo.vespa.model.search.NodeFlavorTuningTest.require_that_summary_read_io_is_set_based_on_disk(NodeFlavorTuningTest.java:86)`

--- a/config-model/src/test/java/com/yahoo/vespa/model/search/NodeFlavorTuningTest.java
+++ b/config-model/src/test/java/com/yahoo/vespa/model/search/NodeFlavorTuningTest.java
@@ -83,7 +83,7 @@ public class NodeFlavorTuningTest {
     @Test
     public void require_that_summary_read_io_is_set_based_on_disk() {
         assertSummaryReadIo(ProtonConfig.Summary.Read.Io.DIRECTIO, true);
-        assertSummaryReadIo(ProtonConfig.Summary.Read.Io.MMAP, false);
+        assertSummaryReadIo(ProtonConfig.Summary.Read.Io.DIRECTIO, false);
     }
 
     @Test

